<Type Name="TransferErrorMessagesTask+DtsTransferErrorMessagesException" FullName="Microsoft.SqlServer.Dts.Tasks.TransferErrorMessagesTask.TransferErrorMessagesTask+DtsTransferErrorMessagesException">
  <TypeSignature Language="C#" Value="public class TransferErrorMessagesTask.DtsTransferErrorMessagesException : Exception" />
  <TypeSignature Language="ILAsm" Value=".class nested public auto ansi serializable beforefieldinit TransferErrorMessagesTask/DtsTransferErrorMessagesException extends System.Exception" />
  <TypeSignature Language="DocId" Value="T:Microsoft.SqlServer.Dts.Tasks.TransferErrorMessagesTask.TransferErrorMessagesTask.DtsTransferErrorMessagesException" />
  <AssemblyInfo>
    <AssemblyName>Microsoft.SqlServer.TransferErrorMessagesTask</AssemblyName>
    <AssemblyVersion>14.0.0.0</AssemblyVersion>
  </AssemblyInfo>
  <Base>
    <BaseTypeName>System.Exception</BaseTypeName>
  </Base>
  <Interfaces />
  <Docs>
    <summary>
            Die Ausnahme, die zur Ladezeit ausgelöst wird, wenn der permanente Name des XML-Stammknotens des Tasks fehlerhaft ist.
            </summary>
    <remarks>Muss hinzugefügt werden.</remarks>
  </Docs>
  <Members />
</Type>
